go.mod

```diff
@@ -0,0 +1,119 @@
+module github.com/influxdata/telegraf
+
+go 1.12
```

Comment:
What is the significance of this line?  Should it be the oldest supported version `go 1.11`?

Reply:
I believe there were some module feature transitions from `go 1.11` to `go 1.12`. Though most have been back ported to 1.11 somewhere around `>= go 1.11.4`. At which point `go 1.11` can parse `go 1.12` safely. So having it at 1.12 should be fine and backwards compatible as far as 1.11.4.

https://github.com/golang/go/issues/30446